### ISSUE TITLE
VP9: Framemarking not usable

### ISSUE THREAD

**Issue by aboba** [juberti/draughts#118]
The following post indicates that framemarking was not usable with VP9: 
https://mailarchive.ietf.org/arch/msg/avt/aOufzlzicXOXpP-1s48IrGUIraU/

Section 5,4 Framemarking does not mention these issues: 
https://tools.ietf.org/html/draft-ietf-payload-vp9-10#page-16

This section should either be updated to suggest a solution to the implementation problems, or it should state that framemarking is not usable.

**Comment by JonathanLennox** [comment]
In my opinion this is an issue with framemarking, that it only supports certain fairly restrictive scalability modes (not including K-SVC), not an issue with the VP9 format.  Does more than this need to be said?

**Comment by juberti** [comment]
I agree with Jonathan, although that suggests a dependency for the vp9 document. Is this on the agenda for the upcoming interim?

**Comment by aboba** [comment]
@juberti Framemarking (implementation experience and WGLC resolution) and VP9 (WGLC comments) are both on the agenda. 
As I understand it, one of the reasons framemarking was removed from Chrome was lack of utility for VP8 & VP9, which raises the question as to why a framemarking section is needed in VP9.  Could you contribute a slide or two to the implementation discussion on the experience with VP8 and VP9?
